.github/workflows/dev-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Release @dev tag to npm
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  release:
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'release dev tag') }}
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
add hash

Reply:
We hash only 3rd party actions (other than GitHub) per guide:

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions